docs/hints-for-use-with-docker.md

```diff
@@ -12,7 +12,18 @@ docker run \
   analyze [...] # Insert further arguments for the command.
 ```
 
-**Note:** The single forward slash `/` between the environment variable `$PWD` and the `:` is required for PowerShell compatibility, as PowerShell otherwise interprets `:` as part of the environment variable. 
+Make sure to mount the project root directory so ORT is able to detect VCS information. To work on a subproject use the option `-i` to specify the actual working directory:
+
+```shell
+docker run \
+  -v $PWD/:/project \ # Mount current working directory into /project to use as input.
+  ort --info \
+  -c /project/ort/config.hocon \ # Use file from "<workingdirectory>/ort" as config.
+  -i /project/subproject \ # Specify working directory
```

Comment:
The `-i` option is specific to the `analyze` sub-command and needs to be moved after it.